cmd/lhsm-plugin-az/main.go

```diff
@@ -24,35 +24,39 @@ import (
 
 type (
 	archiveConfig struct {
-		Name             string `hcl:",key"`
-		ID               int
-		AzStorageAccount string `hcl:"az_storage_account"`
-		AzStorageKey     string `hcl:"az_storage_key"`
-		Endpoint         string
-		Region           string
-		Container        string
-		Prefix           string
-		UploadPartSize   int64  `hcl:"upload_part_size"`
-		NumThreads       int    `hcl:"num_threads"`
-		Bandwidth        int    `hcl:"bandwidth"`
-		MountRoot        string `hcl:"mountroot"`
-		ExportPrefix     string `hcl:"exportprefix"`
-		azCreds          *azblob.SharedKeyCredential
+		Name                  string `hcl:",key"`
+		ID                    int
+		AzStorageAccount      string `hcl:"az_storage_account"`
+		AzStorageKVName       string `hcl:"az_kv_name"`
+		AzStorageKVSecretName string `hcl:"az_kv_secret_name"`
+		AzStorageSAS          string `hcl:"az_storage_sas"`
```

Comment:
From our discussion `az_storage_sas` is no longer needed

Reply:
I'll need that to hold the SAS key after we get it from kv. I'll remove the serialization to get it from config file.